inputFiles/lagrangianContactMechanics/ContactMechanics_PassingCrack_base.xml

```diff
@@ -138,6 +138,13 @@
       scale="0.0"
       setNames="{ rear }"/>
   </FieldSpecifications>
+ 
+  <Functions>
+    <TableFunction
+      name="apertureTable"
+      coordinates="{ -1.0e-3, 0.0 }"
+      values="{ 0.0, 0.0 }"/>
+  </Functions>
```

Comment:
This is not related to your PAMELA/VTK fix, is it?

Reply:
Yes, this is unrelated. `ContactMechanics_PassingCrack` was removed from the integrated tests. As a result, the `xml` file was not updated to ensure that attribute `apertureTableName` was not empty. I decided to modify to make sure it can run.

As a side comment, specifying an aperture makes sense when contact mechanics is coupled with fluid flow. For pure contact mechanics problem this field is irrelevant and should not be a required input.